### DISCUSSION TITLE
[Feature] Group pair of photo-short video in live photo

### DISCUSSION THREAD

**Discussion by NyaomiDEV** [immich-app/immich#9458] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Since my library totally broke when I opted to WebP everything, I decided to start from scratch and reupload all my photo manually. Now, I don't have some (a small number of) Live Photos correctly grouped. They didn't group up because the video had correct metadata while the photo seemingly lacked any. How that is possible, I am not sure.

It would've been awesome if I could've just paired them manually and displayed the metadata from wherever there are metadata to be shown (so in this case it is videos.)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by NicholasFlamy** [comment]
> Since my library totally broke when I opted to WebP everything

How did you do this? What tool?

**Reply by NyaomiDEV** [reply]
I'm referring to the ability to choose WebP thumbnails and previews, introduced some weeks ago.

**Reply by NicholasFlamy** [reply]
Okay, so that's irrelevant other than reasoning to redo your immich instance, got it (just clarifying). So now you're trying to take the photos from the old immich instance and import then into the new one. But you're having trouble with the Live Photos right? 

Also, do you have a backup?

**Reply by NyaomiDEV** [reply]
What happened was that I opted in to WebP format for image previews and thumbnails. Once I ran the thumbnail job for all thumbnails, I noticed some files were left behind, so I deleted them and then I reported issues #8981 and #8982 (I admit some fault is my own). Those have been closed with more checks and fixes; however, my instance remained broken (600+ offline paths). I decided to re-run the thumbnails job just to be super safe but it only made the situation worse with 2000+ offline paths. So I just called it quits: I wiped the Docker stack data, reconfigured Immich and reuploaded every photo from a backup I had already. This made me realize that some photos were just ordered by filename, so I fixed that with exiftool and I re-imported them; for some photos with the same name as the live video (.MOV) they just never paired up in the "new" instance. I noticed that, indeed, the name was the same but the metadata was in the video instead of the photo.

**Reply by NicholasFlamy** [reply]
Huh, okay. Thank you for all of that information. I do remember some talk in a PR I think about live photo handling but I think it had to do with downloading from the server. I'm not completely sure how to handle this. I'll post on the Discord to hopefully get someone more knowledgeable to respond.

**Comment by bo0tzz** [comment]
This is now possible.
